docs/en/latest/plugin-develop.md

```diff
@@ -103,7 +103,7 @@ local _M = {
 }
 ```
 
-Note : The priority of the new plugin cannot be the same as the priority of any existing plugin. In addition, plugins with a high priority value will be executed first in a given phase (see the definition of `phase` in [choose-phase-to-run](#choose-phase-to-run)). For example, the priority of example-plugin is 0 and the priority of ip-restriction is 3000. Therefore, the ip-restriction plugin will be executed first, then the example-plugin plugin.
+Note : The priority of the new plugin cannot be the same as the priority of any existing plugin, you can use the `/v1/schema` method of [control API](./control-api.md) to view the priority of all plugins. In addition, plugins with a high priority value will be executed first in a given phase (see the definition of `phase` in [choose-phase-to-run](#choose-phase-to-run)). For example, the priority of example-plugin is 0 and the priority of ip-restriction is 3000. Therefore, the ip-restriction plugin will be executed first, then the example-plugin plugin.
```

Comment:
Some typos.

> The priority of the new plugin cannot be the same as the priority of any existing plugin

Change to "The priority of the new plugin cannot be same to any existing ones".

> plugins with a high priority

Change to "plugins with higher priority".

Reply:
updated.